sdk_container/src/third_party/coreos-overlay/coreos-base/common-oem-files/common-oem-files-0.ebuild

```diff
@@ -69,10 +70,19 @@ src_compile() {
         fi
     } >"${T}/oem-release"
 
+	oemid_cmdline="${oemid}"
+
+	# In this specific case, the OEM ID from the oem-release file ('ami')
+	# is different from the OEM ID kernel command line parameter ('ec2')
+	# because some services like Afterburn or Ignition expects 'ec2|aws' value.
+	if [[ "${oemid}" == "ami" ]]; then
+		oemid_cmdline="ec2"
+	fi
```

Comment:
Could you do indentation with four spaces? I know that ebuilds in Gentoo usually use tabs for indentation, maybe that's something we will switch to later.